core/src/epicli/data/common/ansible/playbooks/roles/recovery/tasks/common/upload_via_ssh.yml

```diff
@@ -0,0 +1,78 @@
+---
+# Invoke with (example):
```

Comment:
why commented?

Reply:
This is just some description how to use the task file. Variables need to be set before the task file is included / imported.

Reply:
This particular procedure is unused and has been removed in this commit https://github.com/epiphany-platform/epiphany/pull/1359/commits/3a5cc22f95f1a20676722b7968382f876a90be4e :+1: